packages/astro/src/runtime/server/transition.ts

```diff
@@ -52,7 +52,7 @@ export function renderTransition(
 			}
 		}
 	} else if (animationName === 'none') {
-		sheet.addAnimationRaw('old', 'animation: none; opacity: 0; mix-blend-mode: normal;');
+		sheet.addAnimationRaw('old', 'animation: none; mix-blend-mode: normal;');
```

Comment:
Hmm, I might be wrong but I from my testing the `opacity: 0` was needed because of stacking when using view transitions (not the fallback). This should be handled by `mix-blend-mode: normal` though.

Reply:
Yeah i wanted to get your opinion on this. I thought it was for that reason. Isn't the reason to do this so you can see the new element which is behind it? But for `none` is that necessary as both elements are the same?

I removed this because it makes it so that in fallback mode the element is invisible until both animations are complete. We might need a mechanism for VT only styles.

Reply:
Changeed it with this: https://github.com/withastro/astro/pull/8258/commits/00dd8ee665cf9a80ba1a66ce22be323604ef3789